uiowa/uiowa
# Uninstall embed, entity_embed

Resolves #5312 

- remove entity embed
- remove embed

# To test

```
ddev blt ds --site=hr.uiowa.edu && ddev drush @hr.local uli node/add/page
```

- There are no import errors when syncing a v2 and a v3 website and the modules are not enabled. Had to import the text formats first before core.extension tries to uninstall the module.
- HR should still be using media library in wysiwyg which we switched out a long time ago. You should be able to select/place/render as usual.
- HR should still be able to use entity browser (featured content) to find and select items for display (no relation but can be confusing based on the name)
- The code that was removed is a hook that only applies to entity_embed and pre-dates the uiowa repo.
- ~~Should probably run a scan to see if there is any remaining `<drupal-entity...` markup still within v2 sites.~~ This stuff hasn't been rendering anyway because it is no longer allowed in filtered_html.

# Follow-up

- Remove from composer.
